src/test/java/xyz/vegaone/easytracking/tag/TagServiceTest.java

```diff
@@ -106,24 +92,19 @@ public void deleteTag(){
     }
 
     @Test
-    public void updateTagTest(){
+    public void updateTagTest() {
         //given
-        Tag tag = new Tag();
-        tag.setBugId(buildAndSaveBug().getId());
-        tag.setTaskId(buildAndSaveTask().getId());
-        tag.setUserStoryId(buildAndSaveUserStory().getId());
-        tag.setName(TAG_NAME);
-        Tag savedTag = tagService.createTag(tag);
+        Tag savedTag = tagService.createTag(buildAndSaveTag());
```

Comment:
`tagService.createTag()` is called twice. Once here, once in the `buildAndSaveTag()` method. I would suggest to keep it in the `buildAndSaveTag()`